packages/@aws-cdk/aws-apigatewayv2-integrations/lib/websocket/mock.ts

```diff
@@ -0,0 +1,27 @@
+import {
+  WebSocketRouteIntegration,
+  WebSocketIntegrationType,
+  WebSocketRouteIntegrationConfig,
+  WebSocketRouteIntegrationBindOptions,
+} from '@aws-cdk/aws-apigatewayv2';
+
+/**
+ * Mock WebSocket Integration
+ */
+export class MockWebSocketIntegration extends WebSocketRouteIntegration {
```

Comment:
According to the naming convention we've established, this class should be called `WebSocketMockIntegration`.